packages/preview/marina-uol-thesis/1.0.0/typst.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "marina-uol-thesis"
+version = "1.0.0"
+authors = ["Kristaps Jurkans <kris.jurkans@gmail.com>"]
+license = "MIT"
+description = "Unofficial thesis template for the University of Lincoln"
+repository = "https://github.com/krisjdev/university-of-lincoln-thesis"
+keywords = ["uol", "lincoln", "university of lincoln", "UoL", "UOL"]
+categories = ["thesis"]
+entrypoint = "template/src/thesis.typ"
```

Comment:
Your package's entrypoint (and all file it depends on) should be outside of the template directory.